src/Integration/LocalServices/IProjectSystemHelperExtensions.cs

```diff
@@ -31,6 +31,7 @@ internal static class IProjectSystemHelperExtensions
         /// </summary>
         public static bool IsKnownTestProject(this IProjectSystemHelper projectSystem, IVsHierarchy vsProject)
         {
+            //TODO: is this relevant for core projects?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)